cime_config/testlist_allactive.xml

```diff
@@ -375,7 +375,7 @@
       <option name="wallclock">  00:30 </option>
     </options>
   </test>
-  <test name="SMS_D_Ld1" grid="f09_g17" compset="B1850" testmods="allactive/cmip6">
```

Comment:
Please revert this change: this is a new test that I added explicitly to test the cmip6 testmod in a B1850 configuration. Was it causing problems?